h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -600,40 +363,67 @@ protected final void scoreAndBuildTrees(XGBoostModel model, DMatrix trainMat, DM
         model._output._training_time_ms = ArrayUtils.copyAndFillOf(model._output._training_time_ms, model._output._ntrees+1, System.currentTimeMillis());
         if (stop_requested() && !timeout()) throw new Job.JobCancelledException();
         if (timeout()) { //stop after scoring
-          if (!scored) doScoring(model, model.model_info()._booster, trainMat, validMat, true, tmpModelDir);
+          if (!scored) {
+            doScoring(model, model.model_info().getBooster(), true, tmpModelDir);
+          }
           _job.update(_parms._ntrees-model._output._ntrees); //finish
           break;
         }
       }
-      doScoring(model, model.model_info()._booster, trainMat, validMat, true, tmpModelDir);
+      doScoring(model, model.model_info().getBooster(), true, tmpModelDir);
+    }
+
+    // Don't start the tracker for 1 node clouds -> the GPU plugin fails in such a case
+    private void startRabitTracker(RabitTracker rt) {
```

Comment:
@michalkurka good point, the script used by RT seems pretty standard so I think just "having Python" should be enough. Here are the imports used by the tracker:

```
from __future__ import absolute_import

import os
import sys
import socket
import struct
import subprocess
import argparse
import time
import logging
from threading import Thread
```